person.rb

```diff
@@ -1,8 +1,13 @@
+require_relative 'base_decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+require_relative 'nameable'
+
 class Person
   attr_accessor :name, :age
   attr_reader :id
 
```

Comment:
- [ ] Based on the requirements, the `Person` class ought to inherit from the `Nameable` class since the `Nameable` will be a parent class. Kindly implement this change. 👍 😎